### PR TITLE
Update Akamai README.md

### PR DESCRIPTION
**Pull request by GinnyGalicia**
updated markdown

Type of change

- Enhancement

## What does this PR do?

Updated markdown

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/1password/_dev/build/docs/README.md [hunk 1]
```diff
@@ -34,4 +34,4 @@ Uses the 1Password Events API to retrieve information about items in shared vaul
 
 {{fields "item_usages"}}
 
-{{event "item_usages"}}
+{{event "item_usages"}}
```
Comment (efd6) [inline packages/1password/_dev/build/docs/README.md:37]: Please revert this.

Comment (GinnyGalicia) [reply packages/1password/_dev/build/docs/README.md]: Done

**Comment by ellis-elastic** [comment]
Closed and moved under ellis-elastic 3532

**Pull request closed without merging**